content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,181 @@
+
+### Product
+Product Managers(PMs) and designers generally interface with the end-users and stakeholders on a more regular basis that engineering.  As advocates of those individuals, PMs generally tend to "speak the same language" as them.  Their ability to remain fluent in the business language can be relatively simple, however the business language isn't necessarily the same as the ubiquitious language for the team.  Since the ubiquitous language is an agreement with the entire team including the business, there are a few things they must do to remain good stewards of the ubiquitous language for the team.
+
+The first is prematurely translating terms or meanings so their team "understand" it better.  At the end of the day, it will be better to take the time to explain the business domain, terms, and meanings to every member of the team than to try to spare them or yourself.  Words mean things. They also often mean different things to different people with different perspectives.  Using the same terminology as the business, at least at first, will ensure that all perspectives have the same understanding of the underlying meaning of it prior to agreeing upon the correct term or meaning for that thing.
+
+The second thing PMs should avoid doing is being authoritative about the ubiquitous language.  In order for the language to be ubiquitous, _everyone_ must agree on the meaning.  The business, product, design, and engineering.  This means that there is room for negotiation on the terms being used.  Every term in the ubiquitous language will have clear meaning to the business, but that does not mean that it will have a 1:1 correlation (every time) with the terms used in the business.  For example the user of the online shopping business may use the word Shopping Cart consistently, however the team may determine for one reason or another to use the term "Cart" instead.  As long as the business clearly understands that "cart" is the same as "shopping cart" with no ambiguity, then it can used in place of "shopping cart" in the ubiquitious language.
+
+When producing their artifacts, generally in the form of user stories, they should use ubiquitous language continuously and consistently.  An examples story may be:
+
+> As a customer, I want to be able to add products to my shopping cart, so that I can continue shopping until I'm ready to check out
+
+Let's now explore how this user story may make its way into the designs for this application.
+
+### Design
+
+As a frequent communicator with the business, being good stewards of the ubiquitous and business languages applies to deign the same way it does to product.  They additionally have a large artifact that they generate and maintain for use by the team, and that is the visual designs and user interviews. 
```

Comment:
update "deign" in the first sentence to design

Reply:
Last sentence could be more direct. Perhaps: They generate visual designs and insights from research to be used by the team. Ubiquitous language in these artifacts enables the team to use them effectively.